src/couch/test/couchdb_compaction_daemon_tests.erl

```diff
@@ -293,6 +293,13 @@ is_idle(DbName) ->
     {ok, Db} = couch_db:open_int(DbName, [?ADMIN_CTX]),
     Monitors = couch_db:monitored_by(Db),
     ok = couch_db:close(Db),
+    Others = [M || M <- Monitors, M /= self()],
+    if Others == [] -> ok; true ->
+        lists:foreach(fun(Other) ->
+            Args = [Other, process_info(Other)],
```

Comment:
That's quite true. But this is a test case change to try and track down why an assertion is failing. It'll be reverted once we figure that out.